Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -504,16 +505,76 @@ private void OnKeyDown(object sender, KeyEventArgs e)
                         _viewModel.LoadContextMenuCommand.Execute(null);
                         e.Handled = true;
                     }
+                    if (specialKeyState.CtrlPressed)
+                    {
+                        
+                        _settings.WindowSize = _settings.WindowSize + 100;
```

Comment:
I am curious why does this work. I suspect we need to set the property in the viewmodel to trigger change?